packages/react/src/components/MultiSelect/tools/sorting.js

```diff
@@ -9,8 +9,8 @@
  * Use the local `localCompare` with the `numeric` option to sort two,
  * potentially alpha-numeric, strings in a list of items.
  *
- * @param {string} itemA
- * @param {string} itemB
+ * @param {ItemType} itemA
+ * @param {ItemType} itemB
```

Comment:
Needed to resolve TS error [here](https://github.com/carbon-design-system/carbon/pull/14720/files#diff-09c8f869f509702cc938ce57587fbf11794eeb2c4bee3b999dca904eead7bc8cR315-R316)